### PR TITLE
Adding support for ONNX operation Reshape(for onnx models v1.1)

### PR DESCRIPTION
**Pull request by LakshmiKumar23**

### CODE REVIEW

**Comment by kiritigowda** [comment]
Full testing required. With models supported.

**Comment by kiritigowda** [comment]
@LakshmiKumar23 crop layer is merged into the model compiler, pull the latest changes and check if this pull request is ready to be merged?
@hansely can you verify this branch?

**Comment by LakshmiKumar23** [comment]
@kiritigowda Pull the latest changes and tested this branch. Ready to be merged

File: model_compiler/python/nnir.py [hunk 1]
```diff
@@ -357,19 +357,28 @@ def updateLocals(self):
                 elif node.type in ['reshape']:
                     input = self.tensor_dict[node.inputs[0]]
                     param = node.attr.get('shape')
+                    #param = input.shape
```
Comment (kiritigowda) [inline model_compiler/python/nnir.py]: do we need this as a comment?

File: model_compiler/python/nnir.py [hunk 2]
```diff
@@ -357,19 +357,28 @@ def updateLocals(self):
...
                     icount = 1
                     ocount = 1
-                    shape = [input.shape[0]]
+                    #shape = [input.shape[0]]
```
Comment (kiritigowda) [inline model_compiler/python/nnir.py]: do we need this as a comment?

Comment (LakshmiKumar23) [reply model_compiler/python/nnir.py]: Nope. I can get rid of this

File: model_compiler/python/nnir.py (hunk 1 above)
Comment (LakshmiKumar23) [reply model_compiler/python/nnir.py]: This is what needs to be used for ONNX models Version 1.3. I can take it off and add it when I work on version 1.3

Comment (kiritigowda) [reply model_compiler/python/nnir.py]: Yes, add it when we support 1.3 models

**Pull request merged**